### DISCUSSION TITLE
Wrong achievment at Expedition 9 Utopia

### DISCUSSION THREAD

**Discussion by Urogor** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#29] [General]
In the expedition 9, Phase 4, Achievment 3, I have the achievment that appears at image, and its impossible to complete it here. Doing the 7th achievment, you will complete it :)

![No Man's Sky 21_09_2024 21_02_10](https://github.com/user-attachments/assets/451098b8-c350-4ef7-9002-647657d05649)

**Comment by cwmonkey** [comment]
Yeah I could have swapped the order of the milestones, I just didn't want to make a big ugly git commit. What I really should have done is replaced the milestones via patches (and a more logical order), that way the commit would be clean on my site. I may still do that someday.
